app/jobs/socure_docv_results_job.rb

```diff
@@ -17,6 +17,9 @@ def perform(document_capture_session_uuid:)
     #   service_provider_issuer: dcs.issuer,
     # )
 
+    sleep_seconds = IdentityConfig.store.doc_auth_socure_job_sleep_seconds
+    sleep sleep_seconds
```

Comment:
hi @solipet 👋🏿  ... what is the intention of adding a sleep here?

Reply:
Only for testing to simulate a lag in fetching the result. This let's us see the wait page and confirm that the polling works and it either eventually continues on or it times out. We can remove it once we're satisfied.

Reply:
On second thought, I removed this in f42c965098

If we want it for testing, we can add it to the deploy branch.